tests/topotests/bgp_multi_vrf_topo2/test_bgp_multi_vrf_topo2.py

```diff
@@ -35,6 +35,24 @@
 CHAOS_9:
     Verify that all vrf instances fall back
     to backup path, if primary link goes down.
+CHAOS_6:
+    Restart BGPd daemon on DUT to check if all the
+    routes in respective vrfs are reinstalled..
+CHAOS_2:
+    Delete a VRF instance from DUT and check if the routes get
+    deleted from subsequent neighbour routers and appears again once VRF
+    is re-added.
+CHAOS_4:
+    Verify that VRF names are locally significant
+    to a router, and end to end connectivity depends on unique
+    virtual circuits (using VLANs or separate physical interfaces).
+CHAOS_4:
```

Comment:
duplicate?

Reply:
Thanks, It was duplicate, I have removed one.